test/mongorito.test.js

```diff
@@ -31,33 +31,8 @@ const postFixture = require('./fixtures/post');
 */
 
 describe ('Mongorito', function () {
-  it ('expose mongoskin module properties', function () {
```

Comment:
Wasn't removed just to make the suite work, was removed given what was being exposed. I'll re-add a new test based on the mongodb driver needs though, thanks for catching this.